test-requirements.txt

```diff
@@ -0,0 +1,4 @@
+pytest-xdist
+coverage==4.5.4
+pytest-forked
+git+https://github.com/ansible-community/pytest-ansible-units.git
```

Comment:
How does this file get referenced during the test workflow invocation?

Reply:
This file is used by the 'building and installing collection' action to install the python dependencies (runtime and test) of the collection. (https://github.com/ansible-network/github_actions/blob/main/.github/actions/build_install_collection/action.yml#L40-L44)